spec/spec_helper.rb

```diff
@@ -26,12 +26,15 @@ def s3
 
 RSpec.configure do |config|
   def converter
-    @converter ||= if !StaticData::JWT_ENABLE
-                     OnlyofficeDocumentserverConversionHelper::ConvertFileData.new(StaticData.documentserver_url)
-                   elsif StaticData.jwt_data_exist?
-                     OnlyofficeDocumentserverConversionHelper::ConvertFileData.new(StaticData.documentserver_url,
-                                                                                   jwt_key: StaticData.get_jwt_key.strip)
-                   end
+    return @converter if @converter
+
+    return @converter = OnlyofficeDocumentserverConversionHelper::ConvertFileData.new(StaticData.documentserver_url) unless PretestsCheck.jwt_enable?
```

Comment:
I think we can extract those long lines into three methods
```
def no_jwt_converter
OnlyofficeDocumentserverConversionHelper::ConvertFileData.new(StaticData.documentserver_url)
end

def jwt_from_env_converter
OnlyofficeDocumentserverConversionHelper::ConvertFileData.new(StaticData.documentserver_url,
                                                                               jwt_key: ENV.fetch('DOCUMENTSERVER_JWT'))
end

def jwt_from_file_converter
...
```

And simplify `def converter` method to
```
return @converter = no_jwt_converter unless PretestsCheck.jwt_enable? 
return @converter =  jwt_from_env_converter if StaticData...
...
```

And after that I think all rubocop warnings will be fixed